pkg/api/customization/globalrole/validator.go

```diff
@@ -0,0 +1,38 @@
+package globalrole
+
+import (
+	"net/http"
+
+	"github.com/rancher/norman/types"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+)
+
+type Wrapper struct {
+	GlobalRoleLister v3.GlobalRoleLister
+}
+
+func (w Wrapper) Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+	if request.Method != http.MethodPut && request.Method != http.MethodPost {
```

Comment:
can we split this line into 2 checks? As one of them repeats at line 19

Reply:
The check should just be `if request.Method != http.MethodPut`